crates/sui-core/src/authority.rs

```diff
@@ -2511,11 +2512,11 @@ impl AuthorityState {
                             error!("try_create_dynamic_field_info should not fail, {}, new_object={:?}", e, new_object);
                             None
                         }
```

Comment:
Nothing changed here other than formatting? I'd probably remove these changes if you can.